lib/index.js

```diff
@@ -441,8 +441,8 @@ function createRequestValidator(spec, method, route, parameterDefinitions) {
       yield next;
     } catch (err) {
       debug(err);
-
-      this.status = HTTPStatus.BAD_REQUEST;
+      // Temporary : koa-spec should probably not be the global error handler
+      this.status = err.statusCode || HTTPStatus.BAD_REQUEST;
```

Comment:
`status` is koa friendly but Node uses `statusCode` (as well as `express`, `request`, ...) internally. I'd rather stick to the Node version.